### PR TITLE
issue 87

### PR DESCRIPTION
**Pull request by jmastr**

### CODE REVIEW

File: qml/components-v-play/ActionMenuItem.qml [hunk 1]
```diff
@@ -11,5 +11,7 @@ BVApp.MenuItem {
 
     signal clicked
 
-    onSelected: clicked()
+    onSelected: {
+        clicked()
```
Comment (micuintus) [inline qml/components-v-play/ActionMenuItem.qml]: C'mon, you only made this change in another commit to
make the actual fix be a one liner!

Comment (jmastr) [inline qml/components-v-play/ActionMenuItem.qml]: Absolutely right! :D

**Comment by micuintus** [comment]
Oh one question did you add this to the ```ActionMenuItem``` only on purpose?
If you, for example, go to the About page, select one license, go to another menu point and go back to the About page, you can see that it still is in the old state where you left it before.

**Comment by jmastr** [comment]
> So we might want to add this to the MenuItem instead?

Yepp

**Pull request merged**